app/controllers/api/v1/customers_controller.rb

```diff
@@ -8,6 +8,10 @@ class CustomersController < Api::V1::BaseController
       include AddressTransformation
       include ExtraFields
 
+      wrap_parameters :customer, include:
+        Customer.attribute_names +
+        [:billing_address, :shipping_address]
+
```

Comment:
Digging a bit about this automatic wrapping, I saw that it was configurable, so let's use it.